static/css/perdiem.css

```diff
@@ -176,9 +176,21 @@ h2.profile-header {
     font-size: 25px;
 }
 
-p.profile-total {
+h5.profile-total {
     text-align: center;
-    font-size: 25px;
+    font-size: 18px;
+    padding-top: 5px;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-family: Dosis;
```

Comment:
Just realized I don't even need to have font family since it will pull from <h5>